### ISSUE TITLE
Enabling QueuedTracking breaks Provider lookup

### ISSUE THREAD

**Issue by iammattmartin** [piwik/plugin-QueuedTracking#150]
As reported against the Provider plugin, since an upgrade in December, all providers started showing as unknown.

Today we disabled QueuedTracking and provider names started being re-populated again from the geoip database. On re-enabling QT, providers went back to being stored as unknown.

Would be nice to get this fixed and we've been able to replicate on a new install on a new server from latest stable version.

Ref: https://github.com/matomo-org/plugin-Provider/issues/10

**Comment by iammattmartin** [comment]
It appears the "fix" is the same as listed in #136.

Uncommenting this line also fixes the lookup on queuedvisits.

```
[2021-06-01 03:25:36] piwik.DEBUG: GEO: Found IP x.x.x.x location (provider 'geoip2php'): array (   'continent_name' => 'Europe',   'continent_code' => 'EU',   'country_code' => 'GB',   'country_name' => 'United Kingdom',   'city_name' => NULL,   'lat' => 51.496,   'long' => -0.122,   'postal_code' => NULL,   'isp' => 'xxx.',   'org' => 'xxx.', ) [] {"class":"UserCountry","request_id":24586}
```

Although that other issue mentions reverse DNS where as this also stops GeoIP lookups too.

**Comment by AltamashShaikh** [comment]
Closing this in favour of #136
